cocos/network/HttpAsynConnection.m

```diff
@@ -180,8 +180,13 @@ - (BOOL) shouldTrustProtectionSpace:(NSURLProtectionSpace*)protectionSpace
         SecTrustSetExceptions(serverTrust, errDataRef);
         
         SecTrustEvaluate(serverTrust, &trustResult);
+        [(id)errDataRef release];
+        
     }
     
+    [certData release];
+    [(id)certArrayRef release];
```

Comment:
I think there is no need to release `certArrayRef`. Refer https://developer.apple.com/library/mac/documentation/CoreFoundation/Reference/CFArrayRef/index.html#//apple_ref/c/func/CFArrayCreate  The `CFArrayCreate` follows the `Create Rule ownership`. It is returned as a autorelease object. 
So I think you don't need to call release explicit. And you event call release twice.